app/assets/javascripts/mapknitter/Annotations.js

```diff
@@ -4,10 +4,10 @@ MapKnitter.Annotations.include({
     MapKnitter.Resources.prototype.initialize.call(this, options);
 
     var map = options.map,
-        drawOptions = {};
+      drawOptions = {};
 
     this._map = map;
-    this._drawnItems = new L.FeatureGroup().addTo(map);
+    this._drawnItems = = L.featureGroup().addTo(map);
```

Comment:
@DraconianLore you're rebase was perfect! You need to fix an issue here now, you accidentally included 2 spaces. You can commit this suggestion directly if you prefer.

```suggestion
    this._drawnItems = L.featureGroup().addTo(map);
```